std_msgs/Header header

geometry_msgs/Point32 point

float32 ri

geometry_msgs/Pose[] poses




